src/js/components/AlertPanel.js

```diff
@@ -37,27 +47,39 @@ var AlertPanel = React.createClass({
     }
 
     return (
-      <i className={iconClassName}></i>
+      <i className={iconClassName} key="icon"></i>
     );
   },
 
   render() {
-    let classes = classNames('container container-fluid container-pod',
-      'flush-bottom', this.props.className);
+    let classes = classNames(
+      'panel alert-panel text-align-center flush-bottom',
+      this.props.className
+    );
 
     return (
-      <div className={classes}>
-        <Panel ref="panel"
-          className="panel panel-inverse vertical-center horizontal-center
-            text-align-center flush-bottom alert-panel"
-          footer={this.props.footer}
-          footerClass="panel-footer flush-top"
-          heading={this.getIcon()}
-          headingClass="panel-header no-border flush-bottom">
-          {this.getTitle()}
-          {this.props.children}
-        </Panel>
-      </div>
+      <Panel ref="panel"
```

Comment:
Hmmm.. for some reason this takes up the whole screen for me now:
![](https://cl.ly/3l2j110B1n11/Image%202016-10-05%20at%2015.44.55.png)

(I even tried rebasing off of latest `cnvs-upgrade/master`, but that didn't work either)